app/controllers/application_controller.rb

```diff
@@ -32,10 +60,19 @@ def current_user
 
   private
 
-  def render_error_page(status)
-    @page = Page::Error.find_by_http_code!(status)
-    page_template = "pages/#{@page.slug}"
-    template = template_exists?(page_template) ? page_template : 'pages/static'
-    render template, status: status
+  def render_error_page(exception, status, format = params[:format])
```

Comment:
Assignment Branch Condition size for render_error_page is too high. [16.76/15]<br>Method has too many lines. [12/10]